.circleci/config.yml

```diff
@@ -20,15 +20,17 @@ jobs:
           command: |
             python -m venv .venv
             . .venv/bin/activate
+            /home/circleci/project/.venv/bin/python -m pip install --upgrade pip
             pip install -r lynk_up_server/dependencies.txt
       - run:
           name: "Run tests"
           command: |
             . .venv/bin/activate
-            pytest 
+            cd lynk_up_server
```

Comment:
Everyone will have to check their `DJANGO_SETTINGS_MODULE` to make sure the path is not nested